api/openapi-spec/swagger.json

```diff
@@ -4463,12 +4463,28 @@
    "v1.FeatureHyperv": {
     "description": "Hyperv specific features.",
     "properties": {
+     "evmcs": {
+      "description": "EVMCS Speeds up L2 vmexits, but disables other virtualization features. Requires vapic\nDefaults to the machine type setting.\n+optional",
```

Comment:
Default from machine type setting are given automatically by libvirt?

Reply:
Uhm, here I just (mindlessly) copy-pasted the same sentence from all the existing features. I'm not sure this is accurate. Will address this in another PR, I'll keep the sentence for this one.

Reply:
I see. Right, I was just curious